src/vcpkg/vcpkgcmdarguments.cpp

```diff
@@ -181,6 +181,32 @@ namespace vcpkg
         return &loc->second;
     }
 
+    LocalizedString MetadataMessage::format() const
+    {
+        switch (kind)
+        {
+            case MetadataMessageKind::Message: return msg::format(*message);
+            case MetadataMessageKind::Literal: return LocalizedString::from_raw(literal);
+            case MetadataMessageKind::Callback: return callback();
+            case MetadataMessageKind::Unused:
+            default: Checks::unreachable(VCPKG_LINE_INFO);
+        }
+    }
+
+    void MetadataMessage::append_to(LocalizedString& target) const
```

Comment:
```suggestion
    void MetadataMessage::format_to(LocalizedString& target) const
```

(suggestion)

Reply:
I think `format_to` carries the connotation that the target's contents will be destroyed. How about `format_append_to`?

Reply:
Changed to `to_string` since we seem to use that convention elsewhere.